docker/build/build.sh

```diff
@@ -55,27 +55,15 @@ docker build -f ./docker/build/Dockerfile.signer -t orbs:signer .
 # Builds experimental features (extra libraries)
 if [[ $CIRCLE_TAG != v* ]] ;
 then
+    export ORBS_EXPERIMENTAL="true"
+    
     # We use an experimental go.mod for these builds
     rm -rf _bin && mkdir -p _bin _dockerbuild
     rm -f ./_dockerbuild/go.mod.template
     SDK_VERSION=$(cat go.mod | grep orbs-contract-sdk | awk '{print $2}')
     cp ./docker/build/go.mod.template.experimental ./_dockerbuild/go.mod.t
```

Comment:
can we instead of duplicating the logic of creating the go.mod file from template (actually overriding it), just set the `ORBS_EXPERIMENTAL` flag given the `CIRCLE_TAG` value, then choose either the experimental or production template according to the value of `ORBS_EXPERIMENTAL`?